### PR TITLE
Ccs 3842 assembly warning alert

### PR DESCRIPTION
**Pull request by xdavidson**
This PR fixes the wording of Publish Warning for assembly. currently when an assembly failed to publish, the warning message states:
  
  Module versions
  Module failed to publish. Check the following:
     * Are you logged in as a publisher?
     * Does the module have all required metadata?

The new message looks like this:
  Publishing Document
  Assembly failed to publish. Check the following:
     * Are you logged in as a publisher?
     * Does the assembly have all required metadata?

### CODE REVIEW

File: pantheon-bundle/frontend/src/app/versions.tsx [hunk 1]
```diff
@@ -744,6 +744,13 @@ class Versions extends Component<IProps, IState> {
                 })
         }
     }
+
```
Comment (rednitish) [inline pantheon-bundle/frontend/src/app/versions.tsx:747]: I have one query here, as we already know, content type can be either module or assembly, we can compare if content type is one of them other wise return. what do you suggest ?

File: pantheon-bundle/frontend/src/app/versions.tsx [hunk 2]
```diff
@@ -136,13 +136,13 @@ class Versions extends Component<IProps, IState> {
                 {this.state.publishAlertVisible && <div className="notification-container pant-notification-container-md">
                     <Alert
                         variant="warning"
-                        title="Module Versions"
+                        title="Publishing Document"
```
Comment (rednitish) [inline pantheon-bundle/frontend/src/app/versions.tsx:139]: Can we write instead as :

"Publishing" {this.props.contentType}  ?

**Review by rednitish — COMMENTED** [review]
I have few suggestions, let me know, if you differ with my opinion.

File: pantheon-bundle/frontend/src/app/versions.tsx (hunk 2 above)
Comment (xdavidson) [reply pantheon-bundle/frontend/src/app/versions.tsx]: I tried the above suggestions. {this.props.contentType} or {...this.props.contentType} renders empty. I ended up creating a state variable to display the Alert title. That seems to reflect the contentType for publishing alerts.

File: pantheon-bundle/frontend/src/app/versions.tsx [hunk 3]
```diff
@@ -744,6 +747,18 @@ class Versions extends Component<IProps, IState> {
...
     }
+
+    private capitalize = (str) => {
+        if (str === undefined || str.trim().length === 0) {
+            return
+        }
+        return str.charAt(0).toUpperCase() + str.slice(1)
+    }
+
+    private getAlertTitle = () => {
```
Comment (benradey) [inline pantheon-bundle/frontend/src/app/versions.tsx:758]: Could I please ask that this method be renamed? I would expect that a method named "getXXX" is a "safe"/read-only operation, but inside of it you are calling a setState. Maybe "resetAlertTitle" or "determineAlertTitle" or something else if you have a better idea.

File: pantheon-bundle/frontend/src/app/versions.tsx [hunk 4]
```diff
@@ -35,6 +35,7 @@ export interface IProps {
 }
 
 interface IState {
+    alertTitle: any
```
Comment (benradey) [inline pantheon-bundle/frontend/src/app/versions.tsx:38]: Why `any` and not `string`?

File: pantheon-bundle/frontend/src/app/versions.tsx (hunk 3 above)
Comment (xdavidson) [reply pantheon-bundle/frontend/src/app/versions.tsx]: updated

**Comment by benradey** [comment]
Cool, thanks for the updates @xdavidson. I think this looks good, but since @rednitish was the first to comment, I'll let him provide the approving review once he's ready.

File: pantheon-bundle/frontend/src/app/versions.tsx (hunk 2 above)
Comment (rednitish) [reply pantheon-bundle/frontend/src/app/versions.tsx]: Looks cleaner :+1:

**Review by rednitish — APPROVED** [review]
LGTM, Thanks Lisa for quick changes :+1:

**Pull request merged**